RELEASE.md

```diff
@@ -1,4 +1,7 @@
 # Releases #
+## Release IN PROGRESS (2016-11-??) ##
+1. Updated the two custom exceptions to no longer extend the base ```Throwable``` and instead the more restrictive ```RuntimeException```.
```

Comment:
Should be something like:

1. Clean up : Convert '''Throwable''' to more restrictive ```RuntimeException```